R/zchunk_L2233.electricity_water.R

```diff
@@ -629,6 +634,24 @@ module_water_L2233.electricity_water <- function(command, ...) {
       select(LEVEL2_DATA_NAMES[["InputEmissCoeff"]], "efficiency") ->
       L2233.InputEmissCoeff_fut_elecPassthru
 
+    # Recreate liquids limit electricity inputs by cooling techs
```

Comment:
Maybe add a little more info to this comment in case they haven't recently read the electricity file.